File: src/space/cherryband/ari/ui/LookupListener.java
package space.cherryband.ari.ui;

/**
 * Created by itkach on 9/24/14.
 */
public interface LookupListener {
    void onLookupStarted(String query);

    void onLookupFinished(String query);

    void onLookupCanceled(String query);
}

File: src/space/cherryband/ari/ui/LookupListener.kt
package space.cherryband.ari.ui

/**
 * Created by itkach on 9/24/14.
 */
interface LookupListener {
    fun onLookupStarted(query: String?)
    fun onLookupFinished(query: String?)
    fun onLookupCanceled(query: String?)
}